packages/kotti-ui/source/kotti-field-select/KtFieldMultiSelect.vue

```diff
@@ -95,6 +96,7 @@ export default defineComponent({
 	props: {
 		...KOTTI_FIELD_PROPS,
 		...KOTTI_FIELD_MULTI_SELECT_PROPS,
+		maximumSelectable: { default: Number.MAX_SAFE_INTEGER, type: Number },
```

Comment:
this belongs on `KOTTI_FIELD_MULTI_SELECT_PROPS` though

Reply:
Also, I don't agree with the `MAX_SAFE_INTEGER` either for the default value  
this should be the maximum of the options.length which can't be added to the default inline defintion obviously  
so we most likely need a computed `defaultMaximumSelectable` to be calculated always off the `options.length`  as a fallback to `null` default

1) default becomes `() => null` 
2) we have a `defaultMaximumSelectable` computed off options.length
3) what we end up passing to `element-ui` is `props.maximumSelectable ?? defaultMaximumSelectable.value`

Reply:
https://github.com/ElemeFE/element/blob/655d89fe0e0d9c4c7c4ca2541f5cf90437690a76/packages/select/src/option.vue#L74
if u look at this computed from their source code, they disable the code when the `limitIsReached` which should be the options.length actually in our case IMO

Reply:
I also noticed we had the same default for `collapseTagsAfter` and i don't know if i want to change that because this is internal logic from Kotti's side and doesn't matter really if it can be MAX INT at one point

Reply:
alternatively, don't pass it at all if it's not passed to Kotti - so v-bind to `undefined`

Reply:
or ignore this entire thing, ur call - not a blocker 

My fear is future updates in which we will miss a weird side effect from their source code